ogr/services/pagure/pull_request.py

```diff
@@ -191,18 +192,43 @@ def get(project: "ogr_pagure.PagureProject", pr_id: int) -> "PullRequest":
         return PagurePullRequest(raw_pr, project)
 
     @staticmethod
-    def get_files_diff(project: "ogr_pagure.PagureProject", pr_id: int) -> dict:
-        try:
-            return project._call_project_api(
-                "pull-request",
-                str(pr_id),
-                "diffstats",
-                method="GET",
-            )
-        except PagureAPIException as ex:
-            if "No statistics" in ex.pagure_error:
-                return {}
-            raise ex
+    def get_files_diff(
+        project: "ogr_pagure.PagureProject",
+        pr_id: int,
+        retries=0,
+        wait_seconds=3,
+    ) -> dict:
+        """Collect all the changes in the PR and deal with ENOPRSTATS exception.
+        While the PR is changing its status from open to merged
+        exception can be raised by the Pagure service.
+        Wait for a while to see if we can manage to access changes adjusting the
+        retries and wait_seconds parameters.
+
+        """
+        attempt = 0
```

Comment:
I think we should start at the first attempt:

```suggestion
        attempt = 1
```